types/module/module.go

```diff
@@ -476,10 +476,6 @@ func (m *Manager) InitGenesis(ctx sdk.Context, cdc codec.JSONCodec, genesisData
 		}
 	}
 
-	// a chain must initialize with a non-empty validator set
-	if len(validatorUpdates) == 0 {
-		return abci.ResponseInitChain{}, fmt.Errorf("validator set is empty after InitGenesis, please ensure at least one validator is initialized with a delegation greater than or equal to the DefaultPowerReduction (%d)", sdk.DefaultPowerReduction)
-	}
```

Comment:
is there any way we can keep this? this is a really good feedback error for me and many others.

Reply:
That's what we have been trying to discuss since March 23rd, just there's not been a discussion. 

My impression that we probably can, but not without discussing with relevant teams to plan.

I agree with you.